### DISCUSSION TITLE
V400 Theme

### DISCUSSION THREAD

**Discussion by cpasoft** [Guilouz/Klipper-Flsun-Speeder-Pad#76] [General]
Upload the default Speeder Pad theme from the V400.
There is one for SR and the generic for the Speeder Pad but not for V400.

**Comment by Sab-tech-lab** [comment]
![sidebar-background](https://github.com/Guilouz/Klipper-Flsun-Speeder-Pad/assets/54538839/5aba19dc-bce9-4f73-b411-0e1bcd8e0405)

Replace the original file "/home/pi/printer_data/config/.theme/sidebar-background.png" with this modified version.
